packages/api/src/jobs/locales/en/translation.json

```diff
@@ -92,7 +92,7 @@
       "LISTED_IN_PSL": "Listed in the PSL? (Y/N)",
       "NOTES": "Notes",
       "PRODUCT_NAME": "Product name",
-      "QUANTITY": "Quantity ({{unit}})",
+      "QUANTITY": "Quantity",
```

Comment:
Could you please do the same for the other three languages?

Reply:
Thanks for pointing me at this, fixed